Ginger/Ginger/Actions/ApiActionsConversion/ApiActionsConversionWizard.cs

```diff
@@ -0,0 +1,248 @@
+        /// </summary>
+        /// <param name="bf"></param>
+        /// <returns></returns>
+        private bool IsWebServiceTargetApplicationInFlow(BusinessFlow bf)
+        {
+            bool isPresent = false;            
+            foreach (var ta in bf.TargetApplications)
+            {
+                ePlatformType platformType = WorkSpace.Instance.Solution.GetApplicationPlatformForTargetApp(ta.ItemName);
+                isPresent = platformType == ePlatformType.WebServices;
+                if(isPresent)
+                {
+                    break;
+                }
+            }
+            return isPresent;
+        }
+
+        /// <summary>
+        /// This is finish method which does the finish the wizard functionality
+        /// </summary>
+        public override void Finish()
+        {
+            base.mWizardWindow.Close();
+        }
+
+        /// <summary>
+        /// This method is used to Stop the conversion process in between conversion process
+        /// </summary>
+        public void StopConversion()
+        {
+            mConversionUtils.StopConversion();
+        }
+
+        /// <summary>
+        /// This method is used to convert the action in case of Continue & Re-Convert
+        /// </summary>
+        /// <param name="lst"></param>
+        /// <param name="isReConvert"></param>
+        public async void ProcessConversion(ObservableList<BusinessFlowToConvert> lst, bool isReConvert)
+        {
+            ProcessStarted();
+            try
+            {
+                ObservableList<BusinessFlowToConvert> flowsToConvert = new ObservableList<BusinessFlowToConvert>();
+                if (isReConvert)
+                {
+                    ObservableList<BusinessFlowToConvert> selectedLst = new ObservableList<BusinessFlowToConvert>();
+                    foreach (var bf in lst)
+                    {
+                        if (bf.IsSelected)
+                        {
+                            bf.BusinessFlow.RestoreFromBackup(true);
+                            bf.ConversionStatus = eConversionStatus.Pending;
+                            bf.SaveStatus = eConversionSaveStatus.Pending;
+                            flowsToConvert.Add(bf);
+                        }
+                    }
+                }
+                else
+                {
+                    flowsToConvert = ListOfBusinessFlow;
+                }
+
+                if (flowsToConvert.Count > 0)
+                {
+                    await Task.Run(() => mConversionUtils.ConvertToApiActionsFromBusinessFlows(flowsToConvert, ParameterizeRequestBody, PullValidations));
+                }
+                mReportPage.SetButtonsVisibility(true);
+            }
+            catch (Exception ex)
+            {
+                Reporter.ToLog(eLogLevel.ERROR, "Error occurred while trying to convert " + GingerDicser.GetTermResValue(eTermResKey.Activities) + " - ", ex);
```

Comment:
for what there is the " - " extention in error message?
the exception details are parsed and shown automatically so no need it.
I see it in multiple Report.ToLog code lines in this class